jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -482,6 +482,9 @@ def vue_set_value(self, data):
     @observe('plugin_opened', 'layer_selected', 'viewer_selected',
              'stretch_hist_zoom_limits')
     def _update_stretch_histogram(self, msg={}):
+        from jdaviz.configs.imviz.plugins.viewers import ImvizImageView
+        from jdaviz.configs.cubeviz.plugins.viewers import CubevizImageView

```

Comment:
since this is likely called a lot, should we avoid having the imports in the methods (since it likely will cost something to at least check if they're already imported?)

Reply:
ooph, I guess we could check `.__class__.__name__` instead 🤔   But I'm also fine leaving it here if you like that better

Reply:
I'll leave it like this, just as we did to [avoid circular imports in app.py](https://github.com/spacetelescope/jdaviz/blob/f7caad081ceca38f16a6f79a1eda79cdca143323/jdaviz/app.py#L1684).